src/entity/user/User.ts

```diff
@@ -122,6 +122,9 @@ export class User {
   @Column({ type: 'text', nullable: true })
   readmeHtml?: string;
 
+  @Column({ type: 'text', nullable: true })
+  acquisitionChannel: string;
```

Comment:
From RFC this should accept enum values eg. `AcquisitionChannel` from https://github.com/dailydotdev/daily-api/pull/1716?

Reply:
It does in the mutation, we try to keep column text afaik for future additions/changes.
(correct me if i'm wrong here)

Reply:
It will still be text in SQL but its good to set as enum for synthetic sugar in TS

Reply:
eg. `UserAction`

Reply:
The JS enum itself is not used anywhere, I thought it might just litter codebase.

Reply:
Yeah, I thought of that too, initially. But I was pushing the simplest thing as possible. As far as I understood, the value of code cleanliness is not the top priority anymore. We should probably draw a line on how much time we want to spend on like you said, a syntactic sugar similar to this.

Reply:
Does adding an enum here complicate your implementation in any way? It should be just one line update setting a type to entity property we already.

Reply:
Not really, I'm just saying, I just pushed the least code necessary that is acceptable for the requirements. So I didn't bother having an additional code footprint.

Reply:
What is the value of that enum though? The schema already validates the possible values. This would be just another code to change in case we add more values to the schema later on. I would prefer keeping it as a string. We already follow that pattern elsewhere in the code.

Reply:
<img width="780" alt="Screenshot 2024-02-27 at 10 05 43 AM" src="https://github.com/dailydotdev/daily-api/assets/36197304/aa149055-039c-4e32-8c79-532a1c90c7f5">

Just to link this as it was already [discussed](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/797376528?focusedCommentId=798720013#comment-798720013) in the RFC

Reply:
@sshanzel and I just synced. Good to go 🚀